src/components/OfflineWithFeedback.js

```diff
@@ -107,6 +90,26 @@ function OfflineWithFeedback(props) {
     const hideChildren = props.shouldHideOnDelete && !isOffline && props.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE && !hasErrors;
     let children = props.children;
 
+    /**
+     * This method applies the strikethrough to all the children passed recursively
+     * @param {Array} children
+     * @return {Array}
+     */
+    const applyStrikeThrough = useCallback(
```

Comment:
Cool, but I would keep that function outside of the component as it was before, just add a second param of `styles` and that would do the trick.

Reply:
I believe that we have to memoize the result as well.